Fluid.MvcViewEngine/Tags/RenderSectionTag.cs

```diff
@@ -3,24 +3,18 @@
 using System.Text.Encodings.Web;
 using System.Threading.Tasks;
 using Fluid.Ast;
+using Fluid.Tags;
 
-namespace Fluid.MvcViewEngine.Statements
+namespace Fluid.MvcViewEngine.Tags
 {
-    public class RenderSectionStatement : Statement
+    public class RenderSectionTag : IdentifierTag
     {
```

Comment:
Oh yes, so there are different kind of `Tag / Block`. So the grammar / parser is updated according to this ...